plasmapy/tests/__init__.py

```diff
@@ -0,0 +1,3 @@
+"""PlasmaPy tests and test helpers."""
```

Comment:
I'll note this will never make it into the documentation since we do not have a `plasmapy.tests` stub file, and I think it should stay that way.